sdk/storage/Azure.Storage.Common/src/Shared/ClientsideEncryption/Models/EncryptionDataSerializer.cs

```diff
@@ -0,0 +1,196 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Text.Json;
+
+namespace Azure.Storage.Cryptography.Models
+{
+    internal static class EncryptionDataSerializer
```

Comment:
Seeing so much static code here is little bit itchy (see comments on the Encryptor/Decryptor in common). However in this case we don't seem to have class dependencies and we don't have DI container and it's how we do serialization/deserialization everywhere else, so I'm going to pass on this here.